src/components/BenefitPlanHeadPanel.js

```diff
@@ -0,0 +1,171 @@
+                            itemQueryIdentifier="bfName"
+                            action={benefitPlanNameValidationCheck}
+                            //autoFocus={autoFocus}
+                            clearAction={benefitPlanNameValidationClear}
+                            setValidAction={benefitPlanNameSetValid}
+                            shouldValidate={v => this.shouldValidateName(v)}
+                            codeTakenLabel="benefitPlan.name.alreadyTaken"
+                            isValid={isBenefitPlanNameValid}
+                            isValidating={isBenefitPlanNameValidating}
+                            validationError={benefitPlanNameValidationError}
+                        />
+                    </Grid>
+                    <Grid item xs={3} className={classes.item}>
+                        <PublishedComponent
+                            pubRef="core.DatePicker"
+                            module="socialProtection"
+                            label="benefitPlan.dateValidFrom"
+                            required
+                            onChange={v => this.updateAttribute('dateValidFrom', v)}
+                            value={benefitPlan?.dateValidFrom}
+                        />
+                    </Grid>
+                    <Grid item xs={3} className={classes.item}>
+                        <PublishedComponent
+                            pubRef="core.DatePicker"
+                            module="socialProtection"
+                            label="benefitPlan.dateValidTo"
+                            required
+                            onChange={v => this.updateAttribute('dateValidTo', v)}
+                            value={benefitPlan?.dateValidTo}
+                        />
+                    </Grid>
+                    <Grid item xs={3} className={classes.item}>
+                        <NumberInput
+                            min={0}
+                            displayZero
+                            module="socialProtection"
+                            label="benefitPlan.maxBeneficiaries"
+                            onChange={v => this.updateAttribute('maxBeneficiaries', v)}
+                            value={benefitPlan?.maxBeneficiaries ?? ""}
+                        />
+                    </Grid>
+                    <Grid item xs={3} className={classes.item}>
+                        <PublishedComponent
+                            pubRef="policyHolder.PolicyHolderPicker"
+                            module="socialProtection"
+                            withNull
+                            onChange={v => this.updateAttribute('holder', v)}
+                            value={benefitPlan?.holder}
+                        />
+                    </Grid>
+                    <Grid item xs={3} className={classes.item}>
+                        <TextAreaInput
+                            module="socialProtection"
+                            label="benefitPlan.json_ext"
```

Comment:
Maybe it could be good to stick to the convention - pascal case. Please transform 'json_ext' -> jsonExt